src/Login.ts

```diff
@@ -151,6 +179,36 @@ export default class Login {
     }
 }
 
+export interface OidcNativeFlow extends ILoginFlow {
```

Comment:
this could do with a doc-comment.

The other LoginFlows correspond to real things in the spec, like the `m.login.sso` definition at https://spec.matrix.org/v1.7/client-server-api/#definition-mloginsso-flow-schema. Is this a sort of psuedo-flow that is internal to the client?